spec/services/concerns/service_pattern_spec.rb

```diff
@@ -1,4 +1,5 @@
 require "rails_helper"
+require "./app/services/concerns/service_pattern" # don't get this from dfe-analytics
```

Comment:
The spec will include the service pattern from the dfe analytics gem without this 🙃

Reply:
🙃

Reply:
@CatalinVoineag @Nitemaeric FYI I've reported the issue so the dfe-analytics team are aware: https://github.com/DFE-Digital/dfe-analytics/issues/136

It's on their radar now. But we'll need this workaround for the time being.

Perhaps it'd be useful to add a comment to this line to reference that GitHub issue?

```suggestion
require "./app/services/concerns/service_pattern" # needed until DFE-Digital/dfe-analytics#136 is fixed
```